csharp/HOLMS.Platform/HOLMS.Platform/Types/Extensions/IAM/Department.cs

```diff
@@ -1,45 +1,29 @@
 ﻿namespace HOLMS.Types.IAM {
     public partial class Department {
-        public Department(DepartmentIndicator id, string name, bool isHousekeeper, bool capStaffMgmt, bool capGroupMgmt,
-            bool capHousekeepingMgmt, bool capAccountingReports, bool capManagementReports,
-            bool capWorkstationParameters, bool capSaveDisplaySettings, bool capLodgeicReset,
-            bool capEventsWaitlistSetup, bool capSystemSettings, bool capYieldManagement,
-            bool capHoldMgmt, bool capGuestBlacklistAddRemove, bool capOverbookRooms,
-            bool capOverrideRates, bool capCallAccountingAdvancedSetup, bool capGiftShopsalesSetup,
-            bool capCreditCardSetup, bool capCreditCardWorksheet, bool capCreditCardCaptureSettle,
-            bool capCreditCardReporting, bool capExpediaQCMgmt, bool capWebSetup) {
+        public Department(DepartmentIndicator id, string name, bool isHousekeeper, bool capGetSupplySnapshot, bool capGetRoomTypes,
```

Comment:
This sucks.

Since we have enums, do you mind using the enumerated actions and rolling this into a dictionary? The possibility for error with over a dozen positional params of the same type is just too high.